pom.xml

```diff
@@ -92,20 +93,29 @@
             <archive>https://dev.eclipse.org/mhonarc/lists/jacc-dev</archive>
        </mailingList>
     </mailingLists>
+    
     <scm>
         <connection>scm:git:ssh://git@github.com/eclipse-ee4j/jacc.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/jacc.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jacc</url>
     </scm>
 
+	<dependencies>
+		<dependency>
+			<groupId>javax.servlet</groupId>
```

Comment:
I suppose that it will be changes to jakarta.servlet after it's released.

Reply:
I think so too. Added it as comment in the pom.